bin/ycsb

```diff
@@ -51,63 +51,63 @@ COMMANDS = {
 }
 
 DATABASES = {
-    "accumulo"     : "com.yahoo.ycsb.db.accumulo.AccumuloClient",
-    "accumulo1.6"     : "com.yahoo.ycsb.db.accumulo.AccumuloClient",
-    "accumulo1.7"     : "com.yahoo.ycsb.db.accumulo.AccumuloClient",
-    "accumulo1.8"     : "com.yahoo.ycsb.db.accumulo.AccumuloClient",
-    "aerospike"    : "com.yahoo.ycsb.db.AerospikeClient",
-    "arangodb"     : "com.yahoo.ycsb.db.arangodb.ArangoDBClient",
-    "arangodb3"     : "com.yahoo.ycsb.db.arangodb.ArangoDBClient",
-    "asynchbase"   : "com.yahoo.ycsb.db.AsyncHBaseClient",
-    "azurecosmos" : "com.yahoo.ycsb.db.AzureCosmosClient",
-    "azuretablestorage" : "com.yahoo.ycsb.db.azuretablestorage.AzureClient",
+    "accumulo"     : "com.yahoo.ycsb.db.voltdb.accumulo.AccumuloClient",
```

Comment:
none of these other datastores should be moved into a voltdb specific package.

Reply:
Fixed.